package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "blockchain-wallet-v4",
+  "name": "blockchain-wallet-v4-base",
```

Comment:
@schnogz the base package.json and the package.json in `/packages/blockhain-wallet-v4` had the same name and this caused a name conflict error in yarn. Can you see any issues coming up in the ci with changing the base package name?

Reply:
hmm, im not sure... will pull it down and test today